appengine-java11/appengine-simple-jetty-main/pom.xml

```diff
@@ -53,4 +55,25 @@
       <version>9.4.14.v20181114</version>
     </dependency>
   </dependencies>
+  <!-- [END gae_java11_server_dependencies] -->
+
+  <build>
+    <plugins>
+      <!-- Exec Maven Plugin provides goals to help execute Main class locally -->
+      <!-- [START gae_java11_exec_plugin] -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
```

Comment:
While it's probably ok, this plugin hasn't been updated in a long while and starts jetty inside the maven jvm.